test_string_calculator.py

```diff
@@ -0,0 +1,43 @@
+from unittest import TestCase
+import unittest
+from string_calculator import StringCalculator
+
+
+class StringCalculatorTest(TestCase):
+    def setUp(self):
+        self.string_calculator = StringCalculator()
+
+    def test_string_calculator_zero(self):
+        test_string = ''
+        expected_number = 0
```

Comment:
also maybe we can rename the naming of the variables
like: expected_number_response

Reply:
Done